src/js/vic-v2/components/CropperControls.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import classNames from 'classnames';
+
+const moveControlClass = ['cropper-control', 'cropper-control-label-container', 'va-button-link'];
+
+const CropperControls = ({
+  disableMoveDown,
+  disableMoveLeft,
+  disableMoveRight,
+  disableMoveUp,
+  moveDown,
+  moveLeft,
+  moveRight,
+  moveUp,
+  narrowLayout,
+  onSliderChange,
+  onSliderMouseDown,
+  onSliderMouseMove,
+  onSliderMouseUp,
+  rotateLeft,
+  rotateRight,
+  sliderMaxValue,
+  sliderMinValue,
+  sliderValue,
+  zoomIn,
+  zoomOut
```

Comment:
Next step: I'm going to move the CropperJS into this file and call this a `CropperController`- it will be responsible for taking UI actions, translating them into commands for the `CropperJS` canvas, and then sending those commands to the canvas. That should clean up `PhotoField` so that it's only concerned with layout and get rid of the need for this giant list of props.